content/python/suspend_virtual_server.py.md

```diff
@@ -0,0 +1,364 @@
+---
+title: "Working with suspend virtual server"
+description: "A few examples on interacting with suspend virtual server."
+date: "2019-04-11"
+classes: 
+    - "SoftLayer_Product_Package"
+    - "SoftLayer_Product_Order"
+    - "SoftLayer_Virtual_Guest"
+    - "SoftLayer_Location"
+tags:
```

Comment:
remove the `package` and `location` tags as I don't think they really fit here. Add a `suspend` tag

Reply:
Done.